src/OffscreenContextCGL.mm

```diff
@@ -39,7 +37,7 @@
   std::ostringstream out;
   out << "GL context creator: Cocoa / CGL\n"
       << "PNG generator: Core Foundation\n"
-      << "OS info: Mac OS X " << majorVersion << "." << minorVersion << "." << bugFixVersion << " (" << name.machine << " kernel)\n"
+      << "OS info: Mac OS X " << osVersion << " (" << name.machine << " kernel)\n"
```

Comment:
Sorry for nitpicking, but should that maybe show `macOS` instead of `Mac OS X`?

Reply:
Grepping shows also one in PlatformUtils-mac.mm.

Reply:
Yeah.  I tried to preserve the style and strings that already existed.
The OS label could go either way (Mac OS X 12.1  or  MacOS 12.1) and be acceptable. It's just informative text, which doesn't get used often (if at all).

Reply:
Right, we can always change it later if someone complains :grinning:.